subpages/curse-of-knowledge/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="subpage">
```

Comment:
It shows why it is awkward to have this "just this one page" selectors, because when will you have several (the reason for using classes) subpages in the same page?